pkg/utils/secrets/static_token_test.go

```diff
@@ -32,7 +32,7 @@ var _ = Describe("Static Token Secrets", func() {
 
 		BeforeEach(func() {
 			staticTokenConfig = &StaticTokenSecretConfig{
-				Name: common.StaticTokenSecretName,
+				Name: kubeapiserverdeployment.StaticTokenSecretName,
```

Comment:
Can you use a different name here to not depend on the new KAPI package?

Reply:
You mean replace the constant with the actual value because it is a test?
Otherwise I wanted to replace the common.StaticTokenSecretName with the constant from the kubeapiserverdeployment package as the token name is defined there.

Reply:
Yeah, I meant to use a completely unrelated string, something like "foo" or whatever. The test is not depending on the KAPI package, so the name here could be anything.